packages/code-du-travail-frontend/src/contributions/extractMdxContentUrl.js

```diff
@@ -0,0 +1,18 @@
+// Extract external content url from Content tag markdown
+const extractMdxContentUrl = markdown => {
+  if (!markdown) return;
+  // Check Content tag exist on markdown
+  const contentRegExp = new RegExp("<s*Content[^](.*?)s*/>", "g");
```

Comment:
should it be possible to do it using only one regexp ?

Reply:
Yes, it's possible, I'll do that

Reply:
I want to keep like this because on the first rule I only test if Content tag exists and on the second rule i only extract the URL that is between the href quote without having to match regex validate URL.